module/submissions/validator/v1.js

```diff
@@ -25,7 +25,13 @@ module.exports = (req) => {
             .isMongoId().withMessage("Invalid solution id");
             req.checkQuery('entityId').exists().withMessage("required entity id")
             .isMongoId().withMessage("Invalid entity id");
-        }   
+        },
+        getCriteriaQuestions: function () {
+            req.checkParams('_id').exists().withMessage("required submission id");
```

Comment:
@deepakr123 - Add validation of whether or not it is a valid mongoID or not.

Reply:
resolved